packages/features/ee/teams/components/AddNewTeamMembers.tsx

```diff
@@ -164,12 +173,12 @@ export const AddNewTeamMembersForm = ({
       <Button
         EndIcon={ArrowRight}
         color="primary"
-        className="mt-6 w-full justify-center"
+        className="w-full justify-center"
         disabled={publishTeamMutation.isLoading}
         onClick={() => {
           publishTeamMutation.mutate({ teamId });
         }}>
-        {t("team_publish")}
+        {t(orgBranding ? "finish" : "team_publish")}
```

Comment:
As subteams don't need to be published, changed the label to be "Finish" in the second and last step of the team creation wizard